frontend/src/pages/SearchBooks.tsx

```diff
@@ -0,0 +1,89 @@
+import React, {useState} from "react";
+import axios from "axios";
+import {BookAPI} from "../Types/BookAPI";
+import {Box, Container, IconButton, TextField, Typography} from "@mui/material";
+import SearchIcon from "@mui/icons-material/Search";
+
+export default function SearchBooks() {
+    const [search, setSearch] = useState("");
+    const [result, setResult] = useState([]);
+
+    const handleChange = (event: { target: { value: React.SetStateAction<string>; }; }) => {
+        setSearch(event.target.value);
+    }
+
+    const handleSubmit = (event: { preventDefault: () => void; }) => {
+        event.preventDefault();
+        axios.get("https://www.googleapis.com/books/v1/volumes?q=" + search + "&key=AIzaSyC2jzd6VDxU8aIhWzQIWNkZQi4boNbx7Pg&maxResults=20")
+            .then(data => {
+                console.log(data.data.items);
```

Comment:
is das Log absichtlich noch hier?

Reply:
Werde später noch mehr Infos aus der API ziehen, so hab ich es schon da und brauch nicht lange suchen.